spec/spec_helper.rb

```diff
@@ -33,8 +32,6 @@
   config.include SurveyorAPIHelpers
   config.include WaitForAjax
 
-  config.treat_symbols_as_metadata_keys_with_true_values = true
-
```

Comment:
I thought this was added, and isn't this true in the rails app?

Reply:
yeah I think we want to keep this for database cleaner

Reply:
The warnings I got said that this was defaulted to true now and so this config wasn't necessary any more.